src/App.tsx

```diff
@@ -21,6 +41,8 @@ export const App: React.FC = () => {
             data-cy="perPageSelector"
             id="perPageSelector"
             className="form-control"
+            defaultValue={perPage}
+            onChange={e => itemQuantityHandler(+e.target.value)}
```

Comment:
Pls consider moving this function into separate const and using as a callback here. Also `+` before `e.target.value` is redundant